src/sources/soundsourceproxy.cpp

```diff
@@ -428,75 +435,90 @@ namespace {
     }
 } // anonymous namespace
 
-void SoundSourceProxy::loadTrackMetadataAndCoverArt(
-        bool withCoverArt,
-        bool reloadFromFile) const {
+void SoundSourceProxy::parseFileTags(int parseOptions) {
     DEBUG_ASSERT(m_pTrack);
 
+    if (!(parseOptions & PARSE_METADATA_AND_COVERART)) {
```

Comment:
I think it is easier to read if you use the single flags here. I was a short moment confused about the "AND"